client/src/components/organisms/CreateTemplateModal.tsx

```diff
@@ -50,7 +50,14 @@ export const CreateTemplateModal: React.FC = () => {
       choices: choiceState,
     });
   };
-
+  const validated = useMemo(
+    () =>
+      agendaTitleState.length > 0 &&
```

Comment:
민주한테 나머지 파일 시킨 리팩토링인데 변수명에서 "State" 다 빼주세요
```suggestion
      agendaTitle.length > 0 &&
```